### DISCUSSION TITLE
CCombo Arrow is missing in Windows 11

### DISCUSSION THREAD

**Discussion by rostup** [eclipse-platform/eclipse.platform.swt#584] [Q&A]
I recently discovered, that all the Dropdown Arrows in our Eclipse RCP Application are Missing, when it is running on Windows 11.  After testing it with a small new Application using more current Eclipse Versions, I discovered, that the arrow comes back, when I use Eclipse 2022-06 mit Swt 3.120.

I would like to find the changes in swt, that cause the arrow to reappear, so that i can migrate them into the swt that we are using in our application. So far I didn't find any Bug that is talking about it. Can anyone point me to the relevant changes?

**Comment by Phillipus** [comment]
Probably this: https://github.com/eclipse-platform/eclipse.platform.swt/commit/43187fa1ce25eb04c7fe417ffd3e1f7bb025577f
